demo/js/allowoutsideclick.js

```diff
@@ -3,20 +3,21 @@ var createFocusTrap = require('../../');
 var container = document.getElementById('allowoutsideclick');
 var trigger = document.getElementById('activate-allowoutsideclick');
 var active = false;
+var allowOutsideClick = true;
 
-var focusTrap = createFocusTrap('#allowoutsideclick', {
-  allowOutsideClick: function(event) {
-    if (event.target === trigger) {
-      return true;
+function initialize() {
+  return createFocusTrap('#allowoutsideclick', {
+    allowOutsideClick: allowOutsideClick,
```

Comment:
This can be shortened to just `allowOutsideClick`. :)

Reply:
Good catch! I had that initially, but there was a parsing error when I ran `npm run lint`:

```
/Users/seanmcp/dev/seanmcp/focus-trap/demo/js/allowoutsideclick.js
  10:22  error  Parsing error: Unexpected token ,

✖ 1 problem (1 error, 0 warnings)
```

I couldn't figure out what was wrong, and it didn't seem worth the stress for a demo.

Reply:
No worries. I haven't updated all the deps and tooling on this repo like I have on the react one so there's probably some babel-eslint thing missing.